openprescribing/media/js/src/analyse-chart.js

```diff
@@ -530,5 +529,110 @@ var analyseChart = {
     this.updateCharts();
   },
 
+  displayTotals: function() {
+    var selected;
```

Comment:
Tiny point but naming this `selectedOrgs` (or indeed `orgSelection`) would aid readability; at first I assumed this was a flag.

Reply:
Tiny but good! I've changed this